packages/components/src/components/form-submit-error-message/form-submit-error-message.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
+import classNames from 'classnames';
 import Icon from '../icon/icon.jsx';
 
-const FormSubmitErrorMessage = ({ message }) => (
-    <div className='dc-form-submit-error-message'>
+const FormSubmitErrorMessage = ({ className, message }) => (
+    <div className={classNames('dc-form-submit-error-message', className)}>
```

Comment:
Do we need PropTypes for this component?

Reply:
Yeah you are right, it's better to add